src/components/List/specs/List.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import List, { ListItem, ListSubheader } from '../src/index';
+import { HouseIcon, TrashIcon } from '../../Icons';
+
+describe(`List tests`, () => {
+  it('has clickable listitems', () => {
+    const props = {
+      onClick: () => {
+        console.log('Clicked');
+      },
+    };
+
+    const spy = cy.spy(props, 'onClick').as('List item click');
+
+    cy.mount(
+      <List>
+        <ListItem primaryText="Primary text" secondaryText="Secondary text" actionType="nav" onClick={props.onClick} />
+      </List>,
+    );
+
+    cy.get('.denhaag-list__item').contains('Primary text');
+    cy.get('.denhaag-list__item').contains('Secondary text');
+    cy.get('.denhaag-list__item')
+      .click()
+      .then(() => expect(spy).to.have.been.calledOnce);
+  });
+
+  it('a11y and snapshots', () => {
+    cy.snapshots(
+      <List subheader={<ListSubheader>subheader</ListSubheader>}>
+        <ListItem primaryText="ListItem" />
+        <ListItem
+          primaryText="Single-line item"
+          actionType="action"
+          onClick={console.log('TrashIcon click')}
+          leftIcon={<HouseIcon color="inherit"></HouseIcon>}
+          rightIcon={<TrashIcon />}
+        />
+        <ListItem
+          primaryText="Single-line item"
+          secondaryText="Secondary text"
+          onClick={console.log('ListItem click')}
+          leftIcon={<HouseIcon color="inherit" />}
+          actionType="nav"
+        />
+      </List>,
+      {},
+      {
+        selector: '.denhaag-list__item',
+        states: {
+          hover: 'denhaag-list__item--hover',
+          focus: 'denhaag-list__item--focus',
+        },
+      },
+      [{ id: 'list', issue: 402 }],
+    );
```

Comment:
The List snapshots are suffering from the #402 list issues with a broken layout. Can we change the viewport size to test them properly? https://docs.cypress.io/api/commands/viewport#Defaults. (Or within one specific test suite: https://docs.cypress.io/api/commands/viewport#Reset-viewport-via-Cypress-config)

Reply:
The issue is in fact not the testing window size (which is wide enough), but the fact that `cy.snapshots` wraps them in the following div: `<div id="wrapper" style="display: inline-block">`. The problem lies within the `display: inline-block`. The `#wrapper` selector is called when creating the screenshots, meaning that always only the actual content of the test is being captured, and not a whole lot of white screen to the right to fill out the testing window.  Due to this the list snapshots look the way they do, but adding `display: block` (or simply removing `inline-block`) will result in all component snapshots filling out the entire testing window (i.e. having a lot of white in them). 

For now I think we should just leave this as is, and create new visual snapshots once this bug is actually fixed.

Reply:
We could technically add some exception where this is not the case, but it feels like a bit of a hassle to add a workaround for a broken component.